com.unity.netcode.gameobjects/Tests/Runtime/Metrics/PacketMetricsTests.cs

```diff
@@ -0,0 +1,65 @@
+#if MULTIPLAYER_TOOLS
+using System;
+using System.Collections;
+using System.Linq;
+using NUnit.Framework;
+using Unity.Collections;
+using Unity.Multiplayer.Tools.MetricTypes;
+using Unity.Netcode.RuntimeTests.Metrics.Utility;
+using UnityEngine;
+using UnityEngine.TestTools;
+
+namespace Unity.Netcode.RuntimeTests.Metrics
+{
+    internal class PacketMetricsTests
+    {
+        [UnityTest]
+        public IEnumerator TrackPacketSentMetric()
+        {
+            MultiInstanceHelpers.Create(1, out var server, out var clients, 60, MultiInstanceHelpers.InstanceTransport.UTP);
+
+            server.StartServer();
+            clients[0].StartClient();
+
+            var serverMetrics = (NetworkMetrics)server.NetworkMetrics;
+            var waitForMetricValues = new WaitForMetricValues<PacketEvent>(serverMetrics.Dispatcher, NetworkMetricTypes.PacketEventSent, metric => { return metric.PacketCount > 0; });
```

Comment:
Nit, this lines is a bit long and I think this function call should be split.

Reply:
Also nit - the anonymous function can be reduced to `metric => metric.PacketCount > 0` (ie. remove the braces and semicolon)